src/applications/gi/components/search/InstitutionSearchForm.jsx

```diff
@@ -96,15 +87,15 @@ function InstitutionSearchForm({
             </div>
           )}
 
-          <h2 className="vads-u-font-size--h3">{keywordLabel}</h2>
+          <h2 className="vads-u-font-size--h3">Search by keyword</h2>
 
           {/* prod flag for story BAH-13938 */}
           {environment.isProduction() ? (
             <div>
               {' '}
               <KeywordSearch
                 autocomplete={autocomplete}
-                label={keywordSearchLabel}
+                label="Enter a school, location, or employer name"
```

Comment:
Since this value is used twice, it might have been better to keep the keywordSearchLabel constant

Reply:
Constants are back in.